rero_ils/modules/documents/views.py

```diff
@@ -393,24 +395,41 @@ def create_publication_statement(provision_activity):
 
 
 @blueprint.app_template_filter()
-def get_cover_art(record):
+def get_cover_art(record, save_cover_url=True, verbose=False):
     """Get cover art.
 
-    :param record: record
+    :param isbn: isbn of document
+    :param save_cover_url: save cover url from isbn if no electronicLocator
+                           with coverImage exists
+    :param verbose: verbose
     :return: url for cover art or None
     """
-    # electronic
+    # electronicLocator
     for electronic_locator in record.get('electronicLocator', []):
-        type = electronic_locator.get('type')
-        content = electronic_locator.get('content')
-        if type == 'relatedResource' and content == 'coverImage':
+        e_content = electronic_locator.get('content')
+        e_type = electronic_locator.get('type')
+        if e_content == 'coverImage' and e_type == 'relatedResource':
             return electronic_locator.get('url')
-    # isbn
-    isbn = document_isbn(record.get('identifiedBy', [])).get('isbn')
-    if not isbn:
-        return None
-    res = get_remote_cover(isbn)
-    return res.get('image')
+    # ISBN
+    isbns = []
+    for identified_by in record.get('identifiedBy', []):
+        if identified_by.get('type') == 'bf:Isbn':
+            isbns.append(identified_by.get('value'))
+    for isbn in sorted(isbns):
+        isbn_cover = get_remote_cover(isbn)
+        if isbn_cover and isbn_cover.get('success'):
+            url = isbn_cover.get('image')
+            if save_cover_url:
+                pid = record.get('pid')
+                record_db = Document.get_record_by_pid(pid)
+                record_db.add_cover_url(url=url, dbcommit=True, reindex=True)
+                msg = 'Add cover art url: {url} do document: {pid}'.format(
+                    url=url,
+                    pid=pid
```

Comment:
same remark concerning string format (use simply `f()`)

Reply:
Maybe we can change all the format() into f() ?
I will no change it here, because it is writen mostly like this.